### ISSUE TITLE
patch from vanilla crashes when creating object

### ISSUE THREAD

**Issue by H-MLim** [plugdata-team/plugdata#1240]
hello again. sorry for bothering again but just found another problem on plug. there are no problem at all when creating a new patch from a scratch on plug data, but when I try to add object on a patch which I already made and saved on vanilla, and as soon as I type something, the entire plug data crashes. (but no problem on message, symbol, etc. just for object (Ctrl+1)) do you also experience this? this was occured on both v.0.8.0 and 0.8.1, Windows 11.

+
I found that there aren't any problems when I load the vanilla patch, and save as another file (=making the patch as "plug-data-patch"), and load it. but still it crashes on the original patch from vanilla...

**Comment by timothyschoen** [comment]
Hi, and no problem! Bug reports are always welcome :)

This doesn't happen on macOS, I'll test this on Windows tomorrow.

**Comment by timothyschoen** [comment]
Unfortunately I can't reproduce this bug on Windows. I tried both this one and the helpfile crash bug, but neither happens for me. I hope that that means they are fixed, does it still happen for you?

Otherwise, maybe you can send the patch that causes the crash for you, so we can see if it crashes for me like that.

**Comment by H-MLim** [comment]
hi, for the newest version, fortunately I haven't experience this problem for now. (actually I experience only once last week, but since I had oversaved the new patch, it was OK and couldn't remind what was the problem). I thought you fixed this bug, but if it's not, I think maybe it was the problem I used the older version at that time. so it seems no problem, I'll close this for now and will reopen when I experience the same thing. thanks!
